apollo-router/Cargo.toml

```
@@ -77,7 +77,7 @@ mockall = "0.11.0"
 reqwest = { version = "0.11.9", features = ["json", "stream"] }
 serde_json_bytes = { version = "0.2.0", features = ["preserve_order"] }
 test-log = { version = "0.2.8", default-features = false, features = ["trace"] }
-test-span = "0.1.1"
+test-span = "0.3"
```

We write versions in full so we have a track of what was the last working version

```suggestion
test-span = "0.3.0"
```

---

ok, will open a followup then!